File: Java/Code/Petshop.java
public class Petshop {
    private String ID;
    private String namaProduk;
    private String kategori;
    private int harga;

    public Petshop(){}
    
    public Petshop(String ID, String namaProduk, String kategori, int harga) {
        this.ID = ID;
        this.namaProduk = namaProduk;
        this.kategori = kategori;
        this.harga = harga;
    }

    public String getID() {
        return this.ID;
    }

    public String getnamaProduk() {
        return this.namaProduk;
    }

    public String getkategori() {
        return this.kategori;
    }

    public int getharga() {
        return this.harga;
    }
    
}

File: Python/Code/Petshop.py
class Petshop:
    __ID = ""
    __namaProduk = ""
    __kategori = ""
    __harga = 0

    def __init__(self, ID, namaProduk, kategori, harga):
        self.__ID = ID
        self.__namaProduk = namaProduk
        self.__kategori = kategori
        self.__harga = harga

    def getID(self):
        return self.__ID
    
    def getnamaProduk(self):
        return self.__namaProduk
    
    def getkategori(self):
        return self.__kategori
    
    def getharga(self):
        return self.__harga